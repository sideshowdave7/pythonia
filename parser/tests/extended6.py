@BOB(Marly)
def foo():
 pass

