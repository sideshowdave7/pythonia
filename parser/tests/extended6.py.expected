(program (decorated (decorator BOB Marly) (def (foo) (suite (pass)))))

